Phonebook.Frontend/src/app/pages/dashboard/components/overview/dashboard.component.html

```diff
@@ -135,5 +70,8 @@ <h1 i18n="DashboardComponent|Title of the Bookmarked people section@@DashboardCo
         </span>
       </button>
     </div>
+    <div class="pb-expand">
+      <router-outlet></router-outlet>
```

Comment:
I'm not sure if I like this. The pro is you can save the link to the dashboard (team, bookmarked) 

The contra:
The normal start of the phonebook always end in the team page. Maybe we should think about the the user get what he was looking the last time.

Reply:
Only if he has not bookmarked some colleagues. Apart from that nothing should have changed

Reply:
But if preferred the team view but has also favorites saved? That is not what users expect. 

They expect the same site, like they visit it last time.

Reply:
and they get the same site. Only if they don't have any bookmarks it doesn't make sense to display the page. This will be better with the next commits have look than.